source/MaterialXGenOgsXml/CMakeLists.txt

```diff
@@ -17,11 +19,15 @@ endfunction(assign_source_group)
 assign_source_group("Header Files" ${materialx_header})
 assign_source_group("Source Files" ${materialx_source})
 
-add_library(MaterialXGenOgsXml STATIC 
-    ${materialx_source} 
+add_library(MaterialXGenOgsXml SHARED 
```

Comment:
Why does this need to be shared. Building shared should be an option.

Reply:
The final version has neither static nor shared, which means it will respect the MATERIALX_BUILD_SHARED_LIBS flag.
It had to be as shared as the GenGlsl library because it needs to overload SurfaceNodeGlsl in SurfaceNodeMaya.